inference-engine/src/readers/ir_reader/ie_ir_reader.hpp

```diff
@@ -25,7 +27,9 @@ namespace ngraph {
 class Function;
 }  // namespace ngraph
 
-namespace InferenceEngine {
+namespace InferenceEngineIRReader {
+
+INFERENCE_PLUGIN_API(StatusCode) CreateReader(IReader*& reader, ResponseDesc *resp) noexcept;
```

Comment:
why do we need this declaration inside hpp?

Reply:
No, i can remove it